lms/djangoapps/instructor/views/api_urls.py

```diff
@@ -122,6 +122,9 @@
     url(r'^gradebook$',
         'instructor.views.api.spoc_gradebook', name='spoc_gradebook'),
 
+    url(r'^gradebook/(?P<page>[0-9]+)$',
+        'instructor.views.api.spoc_gradebook', name='spoc_gradebook'),
```

Comment:
I use 2 url declaration because i am getting same issue as http://stackoverflow.com/questions/1842389/reverse-for-with-arguments-and-keyword-arguments-not-found